apps/mark-scan/frontend/src/api_auth.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { QUERY_CLIENT_DEFAULT_OPTIONS } from '@votingworks/ui';
+import { renderHook, waitFor } from '../test/react_testing_library';
+import {
+  ApiClient,
+  ApiClientContext,
+  createApiClient,
+  updateCardlessVoterBallotStyle,
+} from './api';
+
+const queryClient = new QueryClient({
+  defaultOptions: QUERY_CLIENT_DEFAULT_OPTIONS,
+});
+const mockBackendApi: ApiClient = {
+  ...createApiClient(),
+  updateCardlessVoterBallotStyle: jest.fn(),
+};
+
+function TestHookWrapper(props: { children: React.ReactNode }) {
```

Comment:
Nice! I know we haven't historically written tests for the react-query integration, directly (we've just relied on coverage through higher-level tests).

Instead of putting this test in an `api_auth.test.tsx` sub-file, we could add it to the existing `api.test.tsx` file, so that this boilerplate is available for other tests of the react-query integration, should anyone else choose to add any.

Reply:
> i think what i actually want is a glue test for the frontends using useBallotStyleManager to verify the integration with the api client mutation

That makes more sense to me!